tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -11,14 +11,28 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use virt_autotest::common;
 use virt_autotest::esxi_utils;
 use Time::Local;
 
-our $VM_POWER_ON = 'Powered on';
-our $VM_POWER_OFF = 'Powered off';
+my $ssh_vm;
+my $VM_POWER_ON = 'Powered on';
+my $VM_POWER_OFF = 'Powered off';
 
 sub run {
-    my $vm_name = console('svirt')->name;
+    if (check_var('BACKEND', 'svirt')) {
+        my $vm_name = console('svirt')->name;
+        run_tests($vm_name);
+    }
+    elsif (check_var('BACKEND', 'qemu')) {
+        foreach my $vm_name (keys %virt_autotest::common::guests) {
```

Comment:
I am afraid this is NOT the logic for MU. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/virtualization/universal/patch_guests.pm#L43,  which is applied in job https://openqa.suse.de/tests/10497782/modules/patch_guests/steps/1/src. Only the guest with the same version of product with incident, will need to be tested.